content/zh/docs/reference/kubernetes-api/common-definitions/label-selector.md

```diff
@@ -0,0 +1,88 @@
+---
+api_metadata:
+  apiVersion: ""
+  import: "k8s.io/apimachinery/pkg/apis/meta/v1"
+  kind: "LabelSelector"
+content_type: "api_reference"
+description: "标签选择器是对一组资源的标签查询。"
+title: "标签选择器"
+weight: 2
+auto_generated: true
+---
+
+<!--
+api_metadata:
+  apiVersion: ""
+  import: "k8s.io/apimachinery/pkg/apis/meta/v1"
+  kind: "LabelSelector"
+content_type: "api_reference"
+description: "A label selector is a label query over a set of resources."
+title: "LabelSelector"
+weight: 2
+-->
+
+
+`import "k8s.io/apimachinery/pkg/apis/meta/v1"`
+
+标签选择器是对一组资源的标签查询。`matchLabels`和`matchExpressions`的结果是`ANDeD`。一个`empty`标签选择器匹配所有对象。一个`null`标签选择器不匹配任何对象。
+<!-- A label selector is a label query over a set of resources. The result of matchLabels and matchExpressions are ANDed. An empty label selector matches all objects. A null label selector matches no objects.-->
+
+<hr>
+
+- **matchExpressions** ([]LabelSelectorRequirement)
+  
+   `matchExpressions` 是`LabelSelectorRequirement`的列表，要求是`ANDed`。
+   
+   <a name="LabelSelectorRequirement"></a> 
```

Comment:
what's this？

Reply:
i do not really know this label, but  the English docs has it, so i left it here.